assets/js/modules/pagespeed-insights/components/dashboard/DashboardPageSpeed.js

```diff
@@ -332,8 +330,17 @@ export default function DashboardPageSpeed() {
 					/>
 				</div>
 			</header>
+			{ isFetching && (
+				<div className="googlesitekit-pagespeed-widget__refreshing-progress-bar-wrapper">
+					<ProgressBar compress />
+				</div>
+			) }
 
-			<section>
+			<section
+				className={ classnames( {
+					'googlesitekit-pagespeed-widget__refreshing': isFetching,
```

Comment:
Let's also add this class to the `Recommendations` element to make sure recommendations are greyed out as well.